patches/0001-Expose-Encoder.Canonical-and-MarshalCanonical-Handle.patch

```diff
@@ -380,21 +377,15 @@ index fca2a09..aaa79c2 100644
 +	} else {
 +		x.byIndex = f
 +	}
++	newM[t] = x
+ 	fieldCache.value.Store(newM)
```

Comment:
@endophage Blah, you are correct.  At 1am for some reason I kept reading the "u" as "o".  I'll see if I can add to a test that duplicates this particular issue.